src/Microsoft.Dnx.Runtime/Project.cs

```diff
@@ -233,12 +236,25 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
 
             project.Dependencies = new List<LibraryDependency>();
 
+            // Files to be packed along with the project
+            var packInclude = rawProject.ValueAsJsonObject(PackIncludePropertyName);
```

Comment:
I'm doing so, but it means more of the JSON is retained in memory without being parsed down to smaller data structures. I'm not sure that's actually useful. Perhaps with the ProjectReader change we could make the amount of data loaded configurable and then only have dnu pack load this much but that seems like a larger change.

Anyway, I'll lazy it for now and we can discuss a better option later.